test/respondent/respondent-test-data.js

```diff
@@ -1,13 +1,30 @@
 const respondentSample = {
```

Comment:
Have this file some logic? Can we save it on json?

Reply:
yes, we can) I will do it